dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/airflow/AirflowConnector.java

```diff
@@ -95,24 +117,107 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
     out.add(new DumpMetadataTask(arguments, FORMAT_NAME));
     out.add(new FormatTask(FORMAT_NAME));
 
+    ZonedIntervalIterable zonedIntervals = queryRangeFromConnectorArguments(arguments);
+    Pair<ZonedDateTime, ZonedDateTime> dateRange = dateRange(arguments);
+
     // Airflow v1.5.0
-    addFullTable(out, "dag.csv", "select * from dag;");
-    addFullTable(out, "task_instance.csv", "select * from task_instance;");
+    addQueryTask(out, "dag.csv", "select * from dag;");
+    // todo think about multiple files with this.addTableTaskWithIntervals()
```

Comment:
Any bug ticket for this?

Reply:
I would say it should usage experience and not the ticket. once we will have some feedbacks, we will "think about it"

Reply:
We can track it in the ticket for visibility and close it if it will not be relevant in the future.

Reply:
added